zerver/tests/test_users.py

```diff
@@ -1805,12 +1809,13 @@ def test_stream_recipient_info(self) -> None:
         # These tests were written with the old default for
         # enable_online_push_notifications; that default is better for
         # testing the full code path anyway.
-        hamlet.enable_online_push_notifications = False
-        cordelia.enable_online_push_notifications = False
-        othello.enable_online_push_notifications = False
-        hamlet.save()
-        cordelia.save()
-        othello.save()
+        def disable_online_push_notifications(users: List[UserProfile]) -> None:
```

Comment:
Why should this be a function? It isn't use anywhere else. I think just the for loop will suffice.

Reply:
Ah right!

Made the changes.